source/process/feature-release.md

```diff
@@ -181,16 +181,16 @@ Day when Leads and PMs decide which major features are included in the release,
     - Push next RC to acceptance and announce in Town Square with new RC link
```

Comment:
```suggestion
    - Push next RC to acceptance and announce in Town Square with new RC link
```
Is this still announced in Town Square or is it in a different channel?

Reply:
No, it can probably just say ``Cut next Release Candidate and check CI servers running on release branch" (combined with the next bullet point).